cs/build/internal/Common.Internal.targets

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="'$(TargetFrameworkProfile)' != 'Profile78'">
+  <ItemGroup Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' != 'Profile78'">
     <Reference Include="System" />
     <Reference Include="System.Xml" />
   </ItemGroup>
-  <Import Condition="'$(TargetFrameworkProfile)' == 'Profile78'" Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
-  <Import Condition="'$(TargetFrameworkProfile)' != 'Profile78'" Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' == 'Profile78'" Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
+  <Import Condition="'$(UsingMicrosoftNETSdk)' != 'true' and '$(TargetFrameworkProfile)' != 'Profile78'" Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="..\nuget\Common.targets"/>
   <Import Project="Versions.targets" />
   <PropertyGroup>
-    <DocumentationFile>$(OutDir)\$(TargetName).xml</DocumentationFile>
+    <DocumentationFile Condition="'$(UsingMicrosoftNETSdk)' == 'true'">$(BaseOutputPath)\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
+    <DocumentationFile Condition="'$(UsingMicrosoftNETSdk)' != 'true'">$(OutDir)\$(TargetName).xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
     <FileToCopy Include="$(TargetDir)\$(TargetName).pdb" Condition="'$(MSBuildRuntimeType)' != 'Mono' or '$(_DebugFileExt)' == '.pdb'" />
     <FileToCopy Include="$(TargetPath).mdb" Condition="'$(MSBuildRuntimeType)' == 'Mono' and '$(_DebugFileExt)' == '.mdb'" />
```

Comment:
Reminder to check on [whether the mdb stuff is needed anymore](https://github.com/Microsoft/bond/pull/868#discussion_r186242217).